### PR TITLE
US79972: Darken icons to ferrite.

### PR DESCRIPTION
**Pull request by njostonehouse**

### CODE REVIEW

File: dropdown.css [hunk 1]
```diff
@@ -47,7 +47,7 @@
 }
 
 .vui-dropdown-menu > ul > li[role='separator'] {
-  border: 0;
+  border: 1px solid transparent;
```
Comment (njostonehouse) [inline dropdown.css]: I don't think this change was intentional.

Comment (njostonehouse) [inline dropdown.css]: This file was generated by the build script, apparently it was not committed last time the sass was updated.

Comment (dbatiste) [inline dropdown.css]: Yeah, so probably it should be updated.

Comment (dlockhart) [inline dropdown.css]: Agreed

**Pull request merged**